src/test/java/picard/util/LiftoverVcfTest.java

```diff
@@ -279,7 +281,9 @@ public void testVcfSorted(final boolean disableSort) {
 
         //try to open with / without index
         try (final VCFFileReader liftReader = new VCFFileReader(liftOutputFile, !disableSort)) {
-            try (final CloseableIterator<VariantContext> iter = liftReader.iterator()) {
+        	final VCFHeader header = liftReader.getFileHeader();
```

Comment:
please use spaces, not tabs to indent

Reply:
please add an assertion that the original vcf _doesn't_ have a "reference" field as well.